app/src/helpers/requests/lloydGeorgeSearchResult.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+type Args = {
+    nhsNumber: string;
+    baseUrl: string;
+};
+
+export type LloydGeorgeStitchResult = {
+    number_of_files: number;
+    total_file_size_in_byte: number;
+    last_updated: string;
+    presign_url: string;
+};
+
+async function getLloydGeorgeRecord({
+    nhsNumber,
```

Comment:
With these helper requests generally the file name should match the name of the exported function.

Rename file to getLloydGeorgeRecord or vice versa